docs/getting-started/apostrophe-cli.md

```diff
@@ -0,0 +1,168 @@
+# The Apostrophe CLI
+
+As shown [in the previous page](creating-your-first-project.md), ApostropheCMS
+has a command line tool to help take care of potentially repetative tasks such
+as creating projects and modules. Let's take a closer look at some of the
+options. If you aren't interested in using the CLI, simply continue on to the
```

Comment:
I still feel conflicted about things like "previous page" and especially "the next section". It assumes a linear flow, where it may not be. "Previous page" makes sense purely as a link back to a section, but maybe it's more appropriate to say "as shown in getting started" or some effect.

Reply:
(please use the "changes requested" option)